### ISSUE TITLE
Podsheets.com is down

### ISSUE THREAD

**Issue by pallejs** [crablar/podsheets#61]
Since few days i noted that the website podsheets.com is down.

**Comment by crablar** [comment]
Hi @pallejs I did take it down to save on the hosting fees. Were you using it?

**Comment by pallejs** [comment]
Hi @crablar  Thanks for the info. To be honest I just freshly registered and planned to use it the next follow days since i am sitting bored at home. But it's understandable to cut spendings these days. I dont't know how much money you require to run the server. But why don't you asking for money for the service? Or donation button. Podcast and coding is my personal interest but the company i work for also sponsors many open source projects i know. I don't know just some of my thoughts.

**Comment by crablar** [comment]
@pallejs you are correct that this would be possible. At this point, it is more a time/bandwidth issue than anything else.
